modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-algorithms-benchmarking_{context}"]
+= Benchmarking Kopia hashing, encryption, and splitter algorithms
+
+You can run Kopia commands to benchmark the hashing, encryption, and splitter algorithms. Based on the benchmarking results, it is possible to select the most suitable algorithm for your workload.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have installed the Kopia client.
```

Comment:
for line #14 we *could* also note the kopia client is available on the must-gather image and can be setup as Michal demonstrated in the above comment.

```
apiVersion: v1
kind: Pod
metadata:
  name: oadp-mustgather-pod
  labels:
    purpose: user-interaction
spec:
  containers:
  - name: oadp-mustgather-container
    image: registry.redhat.io/oadp/oadp-mustgather-rhel9:v1.4
    command: ["sleep"]
    args: ["infinity"]

```

Reply:
Done.